LICENSE

```diff
@@ -659,3 +659,13 @@ specific requirements.
 if any, to sign a "copyright disclaimer" for the program, if necessary.
 For more information on this, and how to apply and follow the GNU AGPL, see
 <http://www.gnu.org/licenses/>.
+
+EdX Inc. wishes to state, in clarification of the above license terms, that
```

Comment:
Why are we messing with the license files? And where did this additional text specifically come from?

Reply:
For some odd reason we had two LICENSE files, one with `.txt` and one without. This additional text is from my lawyer.

Just kidding, it's copied from the other services (ecommerce/discovery) LICENSE files. I think I did this because I was doing drive-by changes/fixes to files in our root. Sorry if it's a little uncalled for.

Reply:
No problem. Pinging @douglashall to make sure this is okay.

Reply:
I talked with @douglashall and we don't see any issue since this is consistent with our other services.

Reply:
Thanks!